docs/api/datatypes.md

```diff
@@ -280,9 +281,6 @@ Return type is mostly useful for setups that rely on types like GraphQL.
 }
 ```
 
-In the above code the password is stored plainly in the password field so it can be validated, but is never stored in the DB.
-__Aliases:__ NONE
```

Comment:
`NONE` is actually an alias for virtual https://github.com/krmannix/sequelize/blob/datatype-doc-fix/lib/data-types.js#L969

Otherwise LGTM